src/main/java/com/code_intelligence/jazzer/agent/RuntimeInstrumentor.kt

```diff
@@ -57,6 +58,24 @@ class RuntimeInstrumentor(
         protectionDomain: ProtectionDomain?,
         classfileBuffer: ByteArray,
     ): ByteArray? {
+        var pathPrefix = ""
+        if (!Opt.instrumentOnly.isEmpty() && protectionDomain != null) {
+            var outputPathPrefix = protectionDomain.getCodeSource().getLocation().getFile().toString()
+            if (outputPathPrefix.isNotEmpty()) {
+                if (outputPathPrefix.contains(File.separator)) {
+                    outputPathPrefix = outputPathPrefix.substring(outputPathPrefix.lastIndexOf(File.separator) + 1, outputPathPrefix.length)
```

Comment:
No need to specify the second parameter of `substring`, as that's implicitly the end of the string. Whichever version you prefer, though.